assets/js/persistence.js

```diff
@@ -13,7 +13,8 @@ var URL_PARAM_Q_LIMIT = 1300,
         '#translation': 'q',
         '#webpageTranslation': 'qP',
         '#analyzation': 'qA',
-        '#generation': 'qG'
+        '#generation': 'qG',
+        '#spellchecking': 'qS'
```

Comment:
@sushain97 This currently conflicts with [sandbox](https://github.com/goavki/apertium-html-tools/blob/master/assets/js/persistence.js#L17). What should I change this to?

Reply:
I don't really think sandbox needs one...

Reply:
@sushain97 It at least needs something, otherwise errors start popping up and breaks the UI.

Reply:
Or let's just make it `qSand`.